src/main.cpp

```diff
@@ -1,18 +1,34 @@
 #include <Arduino.h>
 #include "Led.h"
+#include "Button.h"
 
-// Define output pin and period of blink
-#define Pinout 1
-#define led_delay 50 // This value x2 is the period in ms
+// Define Pins for Peripherals
+#define Pin_led 33
+#define Pin_button 19
 
-// Create Led instance
-Led led(Pinout);
+#define led_cycle 50 // This value x2 is the period in ms
+#define Baudrate 9600
+
+// Create instances
+Led led(Pin_led);
+Button button(Pin_button);
+
+int button_state;
 
 void setup() 
 {
+  Serial.begin(Baudrate);
 }
 
 void loop() {
```

Comment:
the formatting is not consistent, the bracket in on the next line in the setup loop.

Is there a short-key for auto format that makes everything easier?

Reply:
Fixed by using auto formatting, Google C++ style